src/annotations/components/AnnotationsTab.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import {renderWithRedux} from 'src/mockState'
+import {AnnotationsTab} from 'src/annotations/components/AnnotationsTab'
+import * as reactRedux from 'react-redux'
+
+jest.mock('src/resources/components/GetResources')
+jest.mock('src/annotations/actions/thunks')
+const setup = () => {
+  return renderWithRedux(<AnnotationsTab />)
+}
+
+describe('Annotations Tab Component', () => {
+  const useSelectorMock = jest.spyOn(reactRedux, 'useSelector')
+  jest.spyOn(reactRedux, 'useDispatch')
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('renders with a helpful message if no annotation streams are found', () => {
+    useSelectorMock.mockReturnValue({id: 'blahidorg'})
+    useSelectorMock.mockReturnValue([])
+
+    const {getByTestId} = setup()
+
+    expect(getByTestId('annotations-empty-state')).toBeInTheDocument()
```

Comment:
```suggestion
    expect(getByTestId('annotations-empty-state')).toBeVisible()
```
might there be situations where it's in the document but erroneously not displaying?